src/GTP.cpp

```diff
@@ -218,7 +219,9 @@ bool GTP::execute(GameState & game, std::string xinput) {
         gtp_printf(id, PROGRAM_NAME);
         return true;
     } else if (command == "version") {
-        gtp_printf(id, PROGRAM_VERSION);
+        size_t filename_start = cfg_weightsfile.find_last_of("/\\") + 1;
+        filename_start = filename_start % cfg_weightsfile.length();
```

Comment:
can you use basename(cfg_weightsfile)?

Reply:
basename is not a function in C++11.  It would require pulling in C++17 headers.

Reply:
Correction, it's not part of C++17 either, it's actually part of boost::filesystem.